.github/workflows/sycl-linux-precommit.yml

```diff
@@ -92,6 +92,11 @@ jobs:
             image_options: -u 1001 --device=/dev/dri -v /dev/dri/by-path:/dev/dri/by-path --privileged --cap-add SYS_ADMIN
             target_devices: level_zero:gpu;opencl:gpu
             extra_lit_opts: -j 50
+          - name: E2E tests on Intel Ponte Vecchio GPU for L0 V2
+            runner: '["Linux", "pvc"]'
+            image_options: -u 1001 --device=/dev/dri -v /dev/dri/by-path:/dev/dri/by-path --privileged --cap-add SYS_ADMIN
+            target_devices: level_zero_v2:gpu
```

Comment:
Can we make it such that `level_zero:gpu;level_zero_v2:gpu` would work in a single invocation? If we can fold this into the existing PVC task it would save on job setup and then devops team will be more open to having this run on PVC.

Reply:
Yes, that should work, and that's what I'm planning to do after I make sure this PR works - for now, I just kept it as a separate job to make debugging easier.

Reply:
I would actually prefer to do this in a separate PR. There is a chance that there are still some tests that might fail randomly due to the too old driver version used on CI. Keeping the jobs separate for now could make it easier to deal with them.